bin/opm

```diff
@@ -1509,7 +1510,14 @@ SKIP_INSTALL:
 }
 
 sub get_install_dir () {
-    if (defined $install_into_cwd) {
+    if (defined $install_into_dir) {
+        my $dir = File::Spec->rel2abs($install_into_dir);
+        if (!-d $dir) {
+            make_path $dir;
```

Comment:
I think this is wrong to create a new path here. The user may simply make a typo in the path name. Better report an error instead of creating it.